packages/office-ui-fabric-react/src/components/DetailsList/DetailsColumn.test.tsx

```diff
@@ -47,7 +47,7 @@ describe('DetailsColumn', () => {
   });
 
   it('invokes IColumn#onColumnClick when columnActionMode is ColumnActionsMode.clickable', () => {
-    const column = assign({}, baseColumn, { columnActionsMode: ColumnActionsMode.clickable });
+    const column: IColumn = { ...baseColumn, ...{ columnActionsMode: ColumnActionsMode.clickable } };
```

Comment:
This can be combined with just one spread (same elsewhere):
```suggestion
    const column: IColumn = { ...baseColumn, columnActionsMode: ColumnActionsMode.clickable };
```

Reply:
Good catch!